README.md

```diff
@@ -154,6 +203,57 @@ We enthusiastically welcome contributions and feedback. You should clone the rep
 
 This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
 
+### Clone
+
+MSAL uses a submodule for a "common" library shared by MSAL, ADAL and the Microsoft authentication library
+
+```cmd
+git clone https://github.com/AzureAD/microsoft-authentication-library-for-android.git msal
```

Comment:
Alternate and IMO easier instructions:
`> git clone --recurse-submodules https://github.com/AzureAD/microsoft-authentication-library-for-android.git`